lib/main_containers.pm

```diff
@@ -144,14 +144,14 @@ sub load_host_tests_podman {
     loadtest 'containers/podman_quadlet' unless (is_staging || is_leap("<16") || is_sle("<16") || is_sle_micro("<6.1"));
     # https://github.com/containers/podman/issues/5732#issuecomment-610222293
     # exclude rootless podman on public cloud because of cgroups2 special settings
```

Comment:
Please move this comment just right before the `unless` where it's relevant.

Reply:
thanks, I haven't checked those.